File: 3270-find-the-key-of-the-numbers/3270-find-the-key-of-the-numbers.py
class Solution:
    def generateKey(self, num1: int, num2: int, num3: int) -> int:
        str1 = str(num1).rjust(4, "0")
        str2 = str(num2).rjust(4, "0")
        str3 = str(num3).rjust(4, "0")
        result = ""

        for i in range(0, 4):
            minimum = min(str1[i], str2[i], str3[i])
            result += minimum

        return int(result)
      

File: 3270-find-the-key-of-the-numbers/3270-find-the-key-of-the-numbers.js
/**
 * @param {number} num1
 * @param {number} num2
 * @param {number} num3
 * @return {number}
 */
var generateKey = function(num1, num2, num3) {
  const str1 = String(num1).padStart(4, "0");
  const str2 = String(num2).padStart(4, "0");
  const str3 = String(num3).padStart(4, "0");
  let result = "";
  
  for (let i = 0; i < 4; i++) {
    const min = Math.min(str1[i], str2[i], str3[i]);
    
    result += min;
  }
  
  return Number(result);
};
